tests/pipeline-all-amps/repo/Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM alfresco/alfresco-content-repository:${image.tag}
```

Comment:
Can we just replace the ${image.tag} with latest
This is what is done in the alfresco-with-jolokia and custom-workflow Docker files.
${image.tag} is normally used for the target rather than the source version.